clusterloader2/pkg/framework/framework.go

```diff
@@ -216,3 +244,8 @@ func (f *Framework) ApplyTemplatedManifests(manifestGlob string, templateMapping
 func (f *Framework) isAutomanagedNamespace(name string) (bool, error) {
 	return regexp.MatchString(f.automanagedNamespacePrefix+"-[1-9][0-9]*", name)
 }
+
+func (f *Framework) isStaleAutomanagedNamespace(name string) bool {
+	var namespaceID = regexp.MustCompile(`^test-[a-z0-9]+-[0-9]+$`)
```

Comment:
Make it package-level variable. No need to recompute regex each time we call this function.